go.mod

```diff
@@ -76,6 +76,7 @@ require (
 	go.starlark.net v0.0.0-20240725214946-42030a7cedce
 	go.temporal.io/api v1.39.0
 	go.temporal.io/sdk v1.29.0
+	go.temporal.io/sdk/contrib/opentelemetry v0.6.0
```

Comment:
Please run `go mod tidy` - you also need to remove it as an indirect dep.

Reply:
done